libs/ui-toolkit/src/components/trading-radio-group/radio-group.tsx

```diff
@@ -86,25 +88,36 @@ export const TradingRadio = ({
     'border-vega-clight-700 dark:border-vega-cdark-700'
   );
   return (
-    <label className={wrapperClasses} htmlFor={id}>
-      <RadioGroupPrimitive.Item
-        value={value}
-        className={itemClasses}
-        id={id}
-        data-testid={id}
-        disabled={disabled}
-      >
-        <RadioGroupPrimitive.Indicator className={indicatorClasses} />
-      </RadioGroupPrimitive.Item>
-      <span
-        className={
-          disabled
-            ? 'text-vega-clight-200 dark:text-vega-cdark-200'
-            : 'cursor-pointer'
-        }
-      >
-        {label}
-      </span>
-    </label>
+    <span className="inline-flex gap-2">
+      <label className={wrapperClasses} htmlFor={id}>
+        <RadioGroupPrimitive.Item
+          value={value}
+          className={itemClasses}
+          id={id}
+          data-testid={id}
+          disabled={disabled}
+        >
+          <RadioGroupPrimitive.Indicator className={indicatorClasses} />
+        </RadioGroupPrimitive.Item>
+        <span
+          className={
+            disabled
+              ? 'text-vega-clight-200 dark:text-vega-cdark-200'
+              : 'cursor-pointer'
+          }
+        >
+          {label}
+        </span>
+      </label>
+      {value && value !== 'custom' && (
+        <span className="text-muted">
```

Comment:
I wouldn't add the copy functionality. This should be passed in as children/render prop